examples/psbt_sign_finalize.rs

```diff
@@ -113,14 +112,27 @@ fn main() {
         value: amount * 4 / 5,
     });
 
-    // Generating signatures & witness data
+    // Plan the Transaction using available assets
```

Comment:
Explain what are assets. The overall structure should be as follows:
Explain the spend policy. In this example we want to spend according to the policy....
- We first plan our spend using the following assets. 
- Mention that a particular subset of assets is enough. 


As an overall comment, we don't need to focus too much on code. We want to explain the process of planning.